R/DBConnection.R

```diff
@@ -106,6 +106,39 @@ setGeneric("dbSendQuery",
   valueClass = "DBIResult"
 )
 
+#' @export
+dbBreak <- structure(list(), class = "dbi_break_sentinel")
+
+dbFetchChunked <- function(rs, n, callback) {
```

Comment:
Something like dbProcessChunked() looks like a more suitable name to me. I thought dbFetchChunked() is supposed to fetch incrementally but return a single big data frame consisting of all chunks. (Perhaps dbFetchChunked() can be implemented using dbProcessChunked()?)